azure_arc_servers_jumpstart/oci/ubuntu/terraform/variables.tf

```diff
@@ -0,0 +1,94 @@
+# Declare TF variables
+
+# Sets location of Azure resource group
+variable "azure_location" {
+  type = string 
+  default = "westus2"
+}
```

Comment:
```suggestion
variable "azure_location" {
  type = string 
  default = "westus2"
  description = "Sets location of Azure resource group"
}
```

Reply:
Apply this suggestion to all variables. Use the description filed and not comments.